lib/kramdown_rpf/kramdown.rb

```diff
@@ -152,6 +171,15 @@ def parse_hints
 
       define_parser(:hints, HINTS_PATTERN)
 
+      # Convert Markdown -> :save
+      # @api private
+      def parse_save
+        @src.pos += @src.matched_size
```

Comment:
not familiar with this code - what does this line achieve? (don't say addition!)

Reply:
As I understand it, it allows the parser to keep track of it's location in the file. We are adding methods to `Kramdown::Parser::GFM` at this point & you can see similar code in the kramdown gem here https://github.com/gettalong/kramdown/blob/master/lib/kramdown/parser/gfm.rb#L126

Reply:
More info here https://github.com/gettalong/kramdown/blob/master/lib/kramdown/parser/kramdown.rb#L23-L59

Reply:
👍